src/main/java/net/datafaker/service/RandomService.java

```diff
@@ -116,6 +116,11 @@ public byte[] nextRandomBytes(int numberOfBytes) {
         return randomBytes;
     }
 
+    public <T extends Enum<T>> T nextEnum(Class<T> klass) {
+        T[] enumConstants = klass.getEnumConstants();
```

Comment:
there is already existing method with similar code https://github.com/datafaker-net/datafaker/blob/646a3f91892c8d7d1bc073a6d105c5c51358c429/src/main/java/net/datafaker/providers/base/Options.java#L146-L149

probably it doesn't have obvious name since it is not the first time there is a PR with another method like that..
How about making one of them reusing another?